docs/functionality/persistence.md

```
@@ -0,0 +1,88 @@
+> Note: Validation results  can number in the **millions**, making **indexing** essential for query performance.
+
+> Note: Data Acquisition behavior is expected to change soon related to capturing/storing referenced resources.
+
+## File Storage (Azure Blob Storage)
+
+Used for persistence of large files and submission artifacts.
+
+### Use Cases
+
+- **Submission Service**
+    - Stores **Patient bundles**
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)